### PR TITLE
xhost: new package at 1.0.8

### PR DESCRIPTION
**Pull request by Chexier**
## Description of package
xhost - server access control program for X

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```
/var/db/kiss/installed/xhost/version
/var/db/kiss/installed/xhost/sources
/var/db/kiss/installed/xhost/manifest
/var/db/kiss/installed/xhost/depends
/var/db/kiss/installed/xhost/checksums
/var/db/kiss/installed/xhost/build
/var/db/kiss/installed/xhost/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/xhost.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/xhost
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/xhost/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
```
Comment (dylanaraps) [inline community/xhost/build]: Remove ` \`

Comment (Chexier) [reply community/xhost/build]: Fixed.

**Comment by dylanaraps** [comment]
One package per PR please. :+1:

**Comment by Chexier** [comment]
> One package per PR please. +1

oh, sorry. How do I fix this situation? I forgot to make another branch for nginx.

**Pull request closed without merging**